README.md

```diff
@@ -208,22 +208,22 @@ $ bin/dry-run.rb bundler dependabot/demo --dir="/ruby"
 
 To view logs and stdout from the native package manager helpers, see [debugging native helpers](#debugging-native-package-manager-helpers).
 
-## Debugging Problems
+# Debugging Problems
 
 The first step to debugging is getting the [development environment running](#getting-a-development-environment-running).
 
 Within the development environment, you have two options for simulating a dependency update job:
 You can use the newly-developed [CLI tool](#cli-tool) or the original [Dry-run script](#dry-run-script).
 
-### CLI tool
+## CLI tool
 
 The [Dependabot CLI](https://github.com/dependabot/cli) is a newly-developed tool that incorporates the [GitHub
 Credentials Proxy](#private-registry-credential-management) to more realistically simulate what's happening within
 the Dependabot-at-GitHub service when talking to private registries.
 
 It has a [dedicated debugging guide](https://github.com/dependabot/cli#debugging-with-the-cli), including support for dropping into the Ruby debugger.
 
-### Dry-Run Script
+## Dry-Run Script
```

Comment:
I noticed in the preview these previously didn't _pop_ as much as I wanted when scrolling quickly through the document (we've got a long'ish `README`!) so I made the a little larger.